<launch>
	<arg name="ugv_id" default="1"/>
    <arg name="ugv_name" default="ugv" />   
    <arg name="demo_id" default="1" />
    
	<node if="$(eval demo_id == 1)" pkg="sunray_tutorial" type="ugv_block_pos" name="ugv_block_pos$(arg ugv_id)" output="screen">
		<param name="ugv_id" value="$(arg ugv_id)" />
		<param name="ugv_name" value="$(arg ugv_name)" />
	</node>
        
    <node if="$(eval demo_id == 2)" pkg="sunray_tutorial" type="ugv_circle_vel" name="ugv_circle_vel$(arg ugv_id)" output="screen">
		<param name="ugv_id" value="$(arg ugv_id)" />
		<param name="ugv_name" value="$(arg ugv_name)" />
	</node>

    <node if="$(eval demo_id == 3)" pkg="sunray_tutorial" type="ugv_circle_vel_body" name="ugv_circle_vel_body$(arg ugv_id)" output="screen">
		<param name="ugv_id" value="$(arg ugv_id)" />
		<param name="ugv_name" value="$(arg ugv_name)" />
	</node>

	<node if="$(eval demo_id == 11)" pkg="sunray_tutorial" type="ugv_block_pos.py" name="ugv_block_pos$(arg ugv_id)" output="screen">
		<param name="ugv_id" value="$(arg ugv_id)" />
		<param name="ugv_name" value="$(arg ugv_name)" />
	</node>

	<node if="$(eval demo_id == 12)" pkg="sunray_tutorial" type="ugv_circle_vel.py" name="ugv_circle_vel$(arg ugv_id)" output="screen">
		<param name="ugv_id" value="$(arg ugv_id)" />
		<param name="ugv_name" value="$(arg ugv_name)" />
	</node>
	
	<node if="$(eval demo_id == 13)" pkg="sunray_tutorial" type="ugv_circle_vel_body.py" name="ugv_circle_vel_body$(arg ugv_id)" output="screen">
		<param name="ugv_id" value="$(arg ugv_id)" />
		<param name="ugv_name" value="$(arg ugv_name)" />

</launch>